docs/migrations/web-twig/MIGRATION-v3.md

```diff
@@ -256,6 +257,53 @@ Use:
 
 See [`Tooltip` documentation][tooltip-readme] for more details and examples.
 
+### Tooltip: Hidden by default
+
+The `TooltipPopover` component is now hidden by default.
+Use `isOpen` prop to show the Tooltip on initial render.
+
+#### Migration Guide
+
+Use `isOpen` prop in `TooltipPopover` instead of `UNSAFE_className="is-hidden"` for setting whether it should be opened or closed on initial render.
+
+Instead of:
+
+```twig
+<Tooltip>
+  <a
+    href="#"
+    data-spirit-toggle="tooltip"
+    data-spirit-target="my-tooltip"
+    aria-describedby="my-tooltip"
+  >
+    I have a tooltip
+  </a>
+  <TooltipPopover id="my-tooltip" UNSAFE_className="is-hidden">
+    Hello there!
+  </TooltipPopover>
+</Tooltip>
+```
+
+Use:
+
+```twig
+<Tooltip>
+  <a
+    href="#"
+    data-spirit-toggle="tooltip"
+    data-spirit-target="my-tooltip"
+    aria-describedby="my-tooltip"
+  >
+    I have a tooltip
+  </a>
+  <TooltipPopover id="my-tooltip">
```

Comment:
should the isOpen prop be here?

Reply:
I think no. This should demonstrate that there is no need for `UNSAFE_className="is-hidden"` and it can be remove.